app/scripts/controllers/preferences-controller.ts

```diff
@@ -925,11 +935,14 @@ export class PreferencesController extends BaseController<
     preference: keyof Preferences,
     value: Preferences[typeof preference],
   ): Preferences {
+    console.log('Setting preference:', { preference, value });
```

Comment:
Once the PR is not in Draft anymore, can you please remove all the console.log calls you added from all files?